docs/integrations/sources/apify-dataset.md

```diff
@@ -0,0 +1,39 @@
+# Apify dataset
+
+## Overview
+
+
+The Apify dataset connect supports full refresh sync only.
+
+It can sync data from a single [Apify Dataset](https://docs.apify.com/storage/dataset) by its ID.
+
+### Output schema
+
+Since the dataset items do not have strongly typed schema, they are synced as objects, without any assumption on their content.
+
+### Features
+
+| Feature | Supported? |
+| :--- | :--- |
+| Full Refresh Sync | Yes |
+| Incremental Sync | No |
+
+### Performance considerations
+
+The Apify dataset connector uses [Apify Python Client](https://docs.apify.com/apify-client-python) under the hood and should handle any API limitations under normal usage.
+
+## Getting started
+
+### Requirements
+
+* Dataset Id
+* Amplitude Secret Key
```

Comment:
Since all we need to provide is `dataset id` and `clean` boolean, do we need Amplitude Secret Key as part of setup at all?

Reply:
no, it was left there by error